roapi/src/server/postgres.rs

```diff
@@ -60,9 +69,19 @@ impl<H: RoapiContext> Engine for RoapiContextEngine<H> {
         &mut self,
         statement: &Statement,
     ) -> Result<Self::PortalType, ErrorResponse> {
-        let query = statement.to_string();
-        let df = self.ctx.sql_to_df(&query).await.map_err(df_err_to_sql)?;
-        Ok(DataFusionPortal { df })
+        if RoapiContextEngine::<H>::ignored_statement(statement) {
```

Comment:
[minor] will following cleaner if modify slightly?
```
        let query = if RoapiContextEngine::<H>::ignored_statement(statement) {
            "SELECT 1 WHERE 1 = 2".to_string()
        } else {
            statement.to_string()
        };

        let df = self.ctx.sql_to_df(&query).await.map_err(df_err_to_sql)?;
        Ok(DataFusionPortal { df })
```

Reply:
yeah i agree. I think ideally, we create an empty dataframe in the ignore branch instead of using a workaround query to avoid unnecessary query parsing and execution.